ledger-core/v2/runner/executionevent/event.go

```diff
@@ -181,13 +186,12 @@ func (e CallMethod) ConstructVCallRequest(execution execution.Context) *payload.
 	execution.Sequence++
```

Comment:
Just saw that execution.Context is not pointer, but object :(
It'll lead to worthlessness increment.

Reply:
should be fixed in different PR with test that proves it >_<

Reply:
sure